src/main/java/com/coveros/selenified/services/Request.java

```diff
@@ -20,6 +20,7 @@
 
 package com.coveros.selenified.services;
 
+import com.google.gson.JsonElement;
 import com.google.gson.JsonObject;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.google.gson.JsonObject'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)